docs/data-sources/waf_overviews_qps_timeline.md

```diff
@@ -0,0 +1,77 @@
+---
+subcategory: "Web Application Firewall (WAF)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_waf_overviews_qps_timeline"
+description: |-
+  Use this data source to query the website QPS.
+---
+
+# huaweicloud_waf_overviews_qps_timeline
+
+Use this data source to query the website QPS.
+
+## Example Usage
+
+```hcl
+variable "start_time" {}
+variable "end_time" {}
+
+data "huaweicloud_waf_overviews_qps_timeline" "test" {
+  from = var.start_time
+  to   = var.end_time
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+* `from` - (Required, Int) Specifies the query start time.
+  The format is 13-digit timestamp in millisecond.
+
+* `to` - (Required, Int) Specifies the query end time.
+  The format is 13-digit timestamp in millisecond.
+
+-> The parameters `from` and `to` must be used together.
```

Comment:
Fields `from` and `to` are required fields, so it is necessary to add this comment.